src/main/java/org/broadinstitute/hellbender/engine/spark/GATKRegistrator.java

```diff
@@ -30,6 +32,10 @@ public void registerClasses(Kryo kryo) {
 
         // JsonSerializer is needed for the Google Genomics classes like Read and Reference.
         kryo.register(Read.class, new JsonSerializer<Read>());
+
+        kryo.register(Arrays.asList("some","values").getClass(), new JavaSerializer());
+        kryo.register(Collections.nCopies(2,"").getClass(), new JavaSerializer());
```

Comment:
I don't want to add a dependency just for that.  It's apache 2 license, can we just copy the class into our repo?